src/App.tsx

```diff
@@ -1,14 +1,54 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
+import { Todo } from './types/Todo';
+import { getTodos, getUser } from './api';
 import { TodoModal } from './components/TodoModal';
+import { User } from './types/User';
 import { Loader } from './components/Loader';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filteredTodos, setFilteredTodos] = useState<Todo[]>([]);
+  const [selectedId, setSelectedId] = useState(0);
+  const [selectedUser, setSelectedUser] = useState<User>();
+  const [selectedTodo, setSelectedTodo] = useState<Todo>();
+  const [isLoading, setIsLoading] = useState(true);
+  const [isSelectedLoading, setIsSelectedLoading] = useState(true);
+
+  useEffect(() => {
+    getTodos().then(setTodos);
+    getTodos().then(setFilteredTodos).finally(() => setIsLoading(false));
+  }, []);
+
+  const filterTodos = (newTodos: Todo[]) => {
```

Comment:
checkout pt 3 of checklist about naming and apply this convention in all places in your code:

**Convention**
```
handleEvent
handleSubjectEvent
```